handler/src/main/java/io/netty/handler/ssl/OpenSslServerContext.java

```diff
@@ -330,9 +331,20 @@ public OpenSslServerContext(
             Iterable<String> ciphers, CipherSuiteFilter cipherFilter, ApplicationProtocolConfig apn,
             long sessionCacheSize, long sessionTimeout, ClientAuth clientAuth, String[] protocols, boolean startTls,
             boolean enableOcsp) throws SSLException {
+        this(trustCertCollection, trustManagerFactory, keyCertChain, key, keyPassword,
```

Comment:
Since this is package-private, modify the existing this existing constructor instead of creating a new one.